lib/sshkit/utils.ex

```diff
@@ -3,7 +3,7 @@ defmodule SSHKit.Utils do
 
   def shellescape(value), do: value
 
-  def shellquote(value), do: value
+  def shellquote(value), do: "'#{value}'" # TODO: Proper quoting
```

Comment:
I'm glad you included the :trollface:.

That's exactly the reason why I included the "TODO", to make the Credo checks fail until I've added proper quoting on this branch 😁